builtin/logical/pki/path_config_crl.go

```diff
@@ -85,9 +86,99 @@ the NextUpdate field); defaults to 12 hours`,
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.ReadOperation: &framework.PathOperation{
 				Callback: b.pathCRLRead,
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							"expiry": {
+								Type: framework.TypeString,
```

Comment:
We have TypeDurationSecond, but I don't know if the response is formatted correctly for that always?

Reply:
Agreed, since we have the tests now, would it be possible to try it out with `TypeTime` and `TypeDurationSecond`?

https://github.com/hashicorp/vault/blob/40f599b11cc164804d0cabfdb20664dff84bccee/sdk/framework/openapi.go#L805-L807

https://github.com/hashicorp/vault/blob/40f599b11cc164804d0cabfdb20664dff84bccee/sdk/framework/openapi.go#L785-L787

Reply:
Same comment applies to:

- auto_rebuild_grace_period
- ocsp_expiry
- delta_rebuild_interval
- expiry

Reply:
Sure! I did update this on a few other PRs. So, I forgot to go back and do it here.

Reply:
With regard to this, the user sees these as string, e.g. `15m` and then it gets parsed into a duration when it's handled. Shouldn't it reflect what the user sees or should it be a duration?
https://developer.hashicorp.com/vault/api-docs/secret/pki#expiry